assets/javascripts/kitten/components/form/modal-footer-input/index.js

```diff
@@ -116,7 +109,7 @@ export const ModalFooterInput = React.forwardRef(
           modifier="beryllium"
           fit="icon"
           size="tiny"
-          disabled={isButtonDisabled}
+          disabled={disabled || value.length < 1}
```

Comment:
si on met des espaces ça va pas le rendre enabled? ou on gère ça côté kisskiss?

Reply:
il n'y a pas de test sur le contenu qui est entré, je pense que c'est à gérer côté KissKiss

Reply:
ça vaut peut-être quand même le coup d'ajouter un `trim` ? `value.trim().length < 1`. Je vois pas vraiment de cas où l'on souhaiterait permettre la soumission d'un espace 😛

Reply:
à ce moment-là peut-être que ça vaudrait le coup de mettre le `trim()` plus haut, par exemple là `setValue(e.target.value.trim())`

Reply:
hmm why not, disons que je trouve ça un peu moins explicite 🤔

Reply:
au moins ça permet de ne ne pas avoir à retraiter aussi dans l’app KissKiss (que ça soit back ou front)